src/features/auth/components/ConfirmChangeEmailForm.tsx

```diff
@@ -0,0 +1,64 @@
+import { FC, useEffect } from 'react';
+import { useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { Form } from 'react-bootstrap';
+import * as yup from 'yup';
+import { Constants } from 'utils/constants';
+import { LoadingButton } from 'common/components/LoadingButton';
+
+export type FormData = {
+  verificationCode: number;
+};
+
+type Props = {
+  onSubmit: (data: FormData) => void;
+};
+
+const schema: yup.SchemaOf<FormData> = yup.object().shape({
+  verificationCode: yup.number()
```

Comment:
Need help in fixing the below error when the input value is erased and is null. 
![Screen Shot 2022-01-26 at 4 03 43 PM](https://user-images.githubusercontent.com/20917683/151267431-e7931992-8fa9-4c5d-a99c-a0de832eca28.png)

My thought is to hook into `onChanges` on the field & set the value to `0` when the value is null. But I'm not familiar with what `...register('verificationCode')` does and how to call it from within the `onChanges` event handler.